plugins.go

```diff
@@ -65,6 +65,11 @@ func DescribePlugins() string {
 			others = append(others, s)
 		}
 	}
+
+	for hookPlugin := range eventHooks {
+		others = append(others, hookPlugin)
+	}
+
```

Comment:
Move this above the "Let's alphabetize these" comment and give it their own section: "Event hooks" - you can just print the name as you iterate them. Thanks!